src/Apps/Ingress/Pages/IngressApp/IngressApp.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useReducer } from 'react';
+import {
```

Comment:
did you want to rename this folder to EntryPoints?

Reply:
And does it need a folder for IngressApp? probably smart in case there end up being multiple?

Reply:
I think the IngressApp folder makes sense, but we could remove it later if we consider EntryPoints to be single component folders.